### PR TITLE
Updated Kevin's blog post

### PR DESCRIPTION
**Pull request by kevinlacaille**
Updated the content of the blog post.
- Added video of app in action 
- Embedded app
- Updated copy

Blog compiles as expected.

To Do:
- [ ] Final sweep of the text
- [ ] Perhaps add an additional image or diagram somewhere, if needed

### CODE REVIEW

File: blog/2024-08-29-kevin/index.mdx [hunk 1]
```diff
@@ -0,0 +1,107 @@
+---
+slug: global-vegetation-segmentation
+title: "ML-less Global Vegetation Segmentation at Scale"
+authors: [kevin]
+tags: [computer vision, vegetation analysis, segmentation]
+unlisted: true
+image: https://fused-magic.s3.us-west-2.amazonaws.com/blog-assets/pachama_app.png
```
Comment (kevinlacaille) [inline blog/2024-08-29-kevin/index.mdx:7]: _Note to self_: Need to update this.

File: blog/2024-08-29-kevin/index.mdx [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+hide_table_of_contents: false
+keywords: [computer vision, vegetation analysis, segmentation]
+---
+
+{/*
+import ReactPlayer from 'react-player'
+
+<ReactPlayer className="video__player" playing={true} muted={true} controls height="100%" url="https://fused-magic.s3.us-west-2.amazonaws.com/blog-assets/pachama_land_suitability_app.mp4" width="100%" />
+*/}
```
Comment (kevinlacaille) [inline blog/2024-08-29-kevin/index.mdx:16]: Will be replaced by my own video.

File: src/app-iframe/python/vegetation_segmentation.py
Comment (kevinlacaille) [inline src/app-iframe/python/vegetation_segmentation.py:1]: This code needs updating, but using it as a WIP.

**Comment by kevinlacaille** [comment]
@pgzmnk I think we'll need you to upload the video and screenshot to the appropriate S3 bucket. Please let me know if you'd like a different initial view state for the app and if you'd like me to update the video and screenshot, accordingly.

**Comment by pgzmnk** [comment]
Video has been uploaded to S3. The screenshot is good where you put it.

**Comment by kevinlacaille** [comment]
I feel as though we should link to the UDF in use. Will put through a PR for the UDF later tonight and another one for the link in the blog post. 

cc: @pgzmnk

**Pull request merged**